tests/rptest/tests/random_node_operations_test.py

```diff
@@ -140,14 +140,14 @@ def _setup_test_scale(self):
             # scale test setup
             self.max_partitions = 32
             self.producer_throughput = 20000
-            self.node_operations = 30
+            self.node_operations = 15
```

Comment:
Curious if this is a general problem with RandomNodeOperationsTest or one that is specific to Iceberg. If just Iceberg, I wonder if it makes sense to only tweak this when Iceberg is enabled?